code-challenges.test.js

```diff
@@ -13,18 +18,54 @@
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
 
 // a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
+// describe("multByThree", () => {
+//     it("takes in an array of numbers and returns an array with all the numbers multiplied by 3.", () => {
+//         const numbersArray1 = [6, 7, 8, 9, 10]
+//         const numbers1Output = [18, 21, 24, 27, 30]
+//         const numbersArray2 = [24, 27, 30, 33, 36]
+//         const numbers2Output = [72, 81, 90, 99, 108]
+//         expect(multByThree(numbersArray1)).toEqual (numbers1Output)
+//         expect(multByThree(numbersArray2)).toEqual(numbers2Output)
+//     })
+// })
+
+
+//const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+//const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
+//Pseudocode
+// Create a function called multBythree that takes in an array
+// one parameter - array
+//Iterate over array using .map so that we can modify each value in the array
+//return mapped array
+// const multByThree = (array) => {
+//     return array.map(value => value * 3)
+//}
+
+// Test Suites: 1 passed, 1 total
+// Tests:       1 passed, 1 total
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("evenlyDivByThree", () => {
+    it("takes a object as an argument and decides if the number inside it is evenly divisible by three or not.", () => {
+        const object1 = { number: 15 }
+        const object1Output = object1 %3 === 0
```

Comment:
Tests should not have any logic in them, the logic should be in the function itself.  You can set the object1Output to the expected output given under the test variables.